### PR TITLE
added check to the NTP server

### PR DESCRIPTION
**Pull request by m-mora**

### CODE REVIEW

File: src/system/time/services/ntp.cpp [hunk 1]
```diff
@@ -27,7 +30,28 @@ bool TimeProviderNTP::init()
...
+}
+
+bool TimeProviderNTP::UpdateNtp()
+{
+    uint16 utc;
+    timeClient.forceUpdate();
+    timeClient.update();
+    utc = timeClient.getEpochTime();
+    // 1665179521 =  Friday, 7 October 2022 12:00:00 AM
+    for(uint8 tries = 0; (tries < 5) && (utc < 1665179521); tries++)
```
Comment (Elemeants) [inline src/system/time/services/ntp.cpp]: Please set these values into macros.

File: src/system/time/services/ntp.cpp [hunk 2]
```diff
@@ -27,7 +30,28 @@ bool TimeProviderNTP::init()
...
+    {
+        return false;
+    }
     datetime = getDateTimeFromEpochTime(timeClient.getEpochTime());
     return true;
-}
\ No newline at end of file
+}
+
+bool TimeProviderNTP::UpdateNtp()
```
Comment (Elemeants) [inline src/system/time/services/ntp.cpp]: Update method name to `updateNTP`

File: src/system/time/services/ntp.cpp [hunk 3]
```diff
@@ -27,7 +30,28 @@ bool TimeProviderNTP::init()
...
+{
+    uint16 utc;
+    timeClient.forceUpdate();
+    timeClient.update();
+    utc = timeClient.getEpochTime();
+    // 1665179521 =  Friday, 7 October 2022 12:00:00 AM
+    for(uint8 tries = 0; (tries < 5) && (utc < 1665179521); tries++)
+    {
+        timeClient.update();
```
Comment (Elemeants) [inline src/system/time/services/ntp.cpp]: This update do not have any effect, true?

File: src/system/time/services/ntp.cpp [hunk 4]
```diff
@@ -14,7 +14,10 @@ bool TimeProviderNTP::init()
 {
     logger << LOG_INFO << "Initializing NTP..." << EndLine;
     timeClient.begin();
-    timeClient.update();
+    if(this->UpdateNtp())
+    {
+        return false;
+    }
     return true;
```
Comment (Elemeants) [inline src/system/time/services/ntp.cpp]: You can just return `this->updateNTP()`

Comment (jpreciad) [reply src/system/time/services/ntp.cpp]: Changes implemented were enough to solve the problem?

Comment (jpreciad) [reply src/system/time/services/ntp.cpp]: I was reviewing and realized that the problem still there:
Readed time: 230-73-137T237:210:215Z

Comment (m-mora) [reply src/system/time/services/ntp.cpp]: Yes, we saw it too, working on the fix

File: src/system/time/services/ntp.cpp (hunk 1 above)
Comment (m-mora) [reply src/system/time/services/ntp.cpp]: done

**Comment by m-mora** [comment]
@Elemeants  seems that NTP is working, but I still see problems with the algorithm to turn on/off the relays

**Pull request closed without merging**